### ISSUE TITLE
Brave woke me up with a notification at 4:45am

### ISSUE THREAD

**Issue by jspraul** [brave/brave-browser#13110]
I searched for '[notification -"manual test" sort:comments-desc](https://github.com/brave/brave-browser/issues?q=notification+-%22manual+test%22+sort%3Acomments-desc)' and didn't see anything related in the 3 issues not starting with 'Manual Test Run'.

## Description

Brave initiated a notification on my phone this morning just to wake me up at 4:45am.

## Steps to reproduce

1. Skip the intro to the Brave Android app 
2. Go to sleep

## Actual result

3. Get woken up by a notification initiated by Brave at 4:45am

## Expected result

3. Stay asleep

## Issue reproduces how often

Once so far, not 100% on the cause; guessing it was because I closed the intro that is displayed when loading Brave for the first time after it is installed on the device. I pretty much closed the notification immediately in groggy anger.

## Version/Channel Information:

This was the Play Store version. Sorry, but I'm not interested in installing the other versions and attempting to reproduce the issue.

## Device details

   - Install type ARM
   - Device type Phone
   - Android version 10
   
## Brave version

Brave 1.17.75, Chromium 87.0.4280.88

### Website problems only

N/A

### Additional information

I have disabled all notifications for the Brave Android app.

**Comment by thefallofbecause** [comment]
It is not for an app to say what time a person should or should not receive a notification for something, because every person's lifestyle is different. A few years ago I would have been more than happy with a notification at 5am, but not one at midday, because I was working nights.

This is why phones have do not disturb settings so you can turn off all notifications within certain hours of your own choosing.

**Comment by jspraul** [comment]
Thanks for sharing your opinion in response to mine my friend!

**Comment by bsclifton** [comment]
Thanks for the assist there @thefallofbecause - DnD is what we would recommend too 😄 You should be able to change the per-app notifications also. Thanks! 😄👌

**Comment by jspraul** [comment]
Ok.

FWIW, here is the list of Android Apps apart from Brave that have woken me up in the past several years:

**Comment by bsclifton** [comment]
@jspraul this was a Brave ads notification? And not a regular application notification?

@tmancey are ads presented when the phone is locked?

**Comment by jspraul** [comment]
No, sorry for the confusion. IIRC it was a notification to return to the introductory tutorial/intro pop-up screen... but as I mentioned I don't really remember much, sorry. I should not have said marketing since my interpretation is probably non-standard (something designed pull new users back into the app = marketing to me).

All I'm asking for is for some empathy towards those with family and friends who can't really afford to shut down notifications, and 'ain't got time for that' to setup everything properly for just their contacts, etc. or however it is technically correct to be done.

Keep up the good work!

**Comment by tmancey** [comment]
@bsclifton Ads are only presented when the phone is awake. Thanks

**Comment by tmancey** [comment]
For reference, if the phone is awake and brave is backgrounded on Android. Ads are not shown between 9pm and 6am. If the phone is awake and foregrounded ads can be shown.

**Comment by bsclifton** [comment]
Awesome! Thanks for confirming, @tmancey 😄 Seems it was our app (Brave proper, not ads) that was sending the notification

Should we consider a similar time restriction @anthonypkeane? I'll re-open ticket until we can answer that

**Comment by timchilds** [comment]
@Uni-verse please try to repo.

**Comment by Uni-verse** [comment]
Unable to reproduce on Pixel / Samsung device.
